src/csharp/Grpc.Tools/build/_protobuf/Google.Protobuf.Tools.targets

```diff
@@ -111,7 +111,7 @@
        Protobuf_Compile directly where required. -->
   <!-- TODO(kkm): Do shared compile in outer multitarget project? -->
   <Target Name="_Protobuf_Compile_BeforeCsCompile"
-          BeforeTargets="BeforeCompile"
+          BeforeTargets="CoreCompile"
```

Comment:
qq: would `BeforeTargets="CoreCompile; BeforeCompile"` work too? It looks like it would be a safer option?

Reply:
It might, but it's pointless.  The issue with WPF projects is that it creates a secondary project internally to do the markup compilation.  `CoreCompile` gets called for both this and the outer project, while `BeforeCompile` is only called for the outer project.  This causes compilation of the inner project to fail and/or use outdated code.

As far as I am aware, `CoreCompile` is valid for both Framework and Core, and it happens at the correct time, unlike `BeforeCompile`.

Reply:
I don't know why you're making that big a deal of this.  It is literally the very next target, so it barely changes the time of execution at all.  Except that it actually works.

Reply:
Maybe @kkm000 can weigh in?